src/topology/category/Profinite.lean

```diff
@@ -68,6 +68,71 @@ end Profinite
 @[simps, derive [full, faithful]]
 def Profinite_to_Top : Profinite ⥤ Top := induced_functor _
 
+namespace Profinite
+
+universe u
+
+/-- An explicit limit cone for a functor to `Profinite`. -/
+def limit_cone {J : Type u} [small_category J] (F : J ⥤ Profinite.{u}) : limits.cone F :=
+{ X :=
+  { to_Top := (Top.limit_cone' $ F ⋙ Profinite_to_Top).X,
+    is_compact := begin
+      erw ← compact_iff_compact_space,
```

Comment:
If you don't like the `erw` then `apply compact_iff_compact_space.mp,` suffices here.